### PR TITLE
Don't merge yet -- [CF1] Dash SSO App

### PR DESCRIPTION
**Pull request by kodster28**
Addresses PCX-902.

### CODE REVIEW

**Comment by kodster28** [comment]
localhost_8000_applications_configure-apps_dash-sso-apps (1)

File: products/cloudflare-one/src/content/applications/configure-apps/dash-sso-apps.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+## Set up dashboard SSO
+
+### Step 1 — Launch Cloudflare for Teams
+
+To log into Cloudflare for Teams directly, go to the [Teams dashboard](https://dash.teams.cloudflare.com/) and select your account.
+
+To log in through the Cloudflare dashboard:
+
+1. Log into your account.
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/applications/configure-apps/dash-sso-apps.md]: Should this have a link? I'm not sure it's clear what to log into

File: products/cloudflare-one/src/content/applications/configure-apps/dash-sso-apps.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+1. Select an account and an application.
+1. Navigate to **Access**.
+1. Select **Launch Teams**.
+1. Select your account.
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/applications/configure-apps/dash-sso-apps.md:28]: Also, the first step mentions "log into your account" and this is "select your account". Is it the same?

File: products/cloudflare-one/src/content/applications/configure-apps/dash-sso-apps.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+Configure an IdP following [our detailed instructions](/identity/idp-integration).
+
+### Step 3 — Contact your account team
+
+Ask your account team to approve your SSO domain.
+
+### Step 4 — Customize your SSO application
+
+Once your account team has approved your SSO domain, you will see an **SSO App** in the [Teams dashboard](https://dash.teams.cloudflare.com/) (under **Access** > **Applications**). This application automatically has a rule to *allow email domain* and uses your IdP as the authentication provider.
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/applications/configure-apps/dash-sso-apps.md]: Not sure if this "*allow email domain*" should be "`allow email domain`" instead

**Review by marciocloudflare — APPROVED** [review]

**Pull request merged**